components/core/src/os/process/windows.rs

```diff
@@ -32,6 +32,16 @@ pub fn become_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
     become_child_command(command, args)
 }
 
+/// Get process identifier of calling process.
+pub fn current_pid() -> u32 {
+    unsafe { kernel32::GetCurrentProcessId() as u32 }
+}
+
+/// Determines if a process is running with the given process identifier.
+pub fn is_alive(pid: u32) -> bool {
+    handle_from_pid(pid).is_some()
```

Comment:
@mwrock is that because `handle_from_pid` allocates memory outside of Rust and doesn't handle the cleanup?